js/jquery.main.js

```diff
@@ -41,7 +41,7 @@ function initIframeChange() {
       url += '&collections=' + window.themes + '&noViews=false';
     }
     else {

```

Comment:
You can remove the entire `else { url }` block :) 
Same thing below

Reply:
Done! :)